data/scenario-test.cfg

```diff
@@ -351,6 +351,8 @@ Xu, Xu, Qxu, Qxu, Ql, Ql, Ql, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Gg, Gg, Gg, Gg, Gg
                                 female_name= "female^darkens"
                                 description= "darkens and stuff"
                                 affect_self=yes
+                                [affect_adjacent]
+                                [/affect_adjacent]
                             [/illuminates]
```

Comment:
@gfgtdf retry with that.

Reply:
with that line the white mage is expected to have -25% multipler because it it not effected by its own illuminates.

Reply:
or wait there is also an effect_self of i'll try

Reply:
hmm wait agin, affect adjacent will give the 'darkens' ability to all the adjacent units? what woudl be a usecase for such a code?

Reply:
i had fixed scenario-test and now this function normally, thank you.